### DISCUSSION TITLE
MacOS use of GDB

### DISCUSSION THREAD

**Discussion by GiacomoPaltrinieri** [WebFreak001/code-debug#396] [General]
Hello everyone, my university requires the use of GDB as a debugger for C++. However, I am unable to install it on my MacBook M1.
Is this extension suitable for the use of GDB on my computer?

**Comment by GitMensch** [comment]
As log as your GDB installation:
* works outside of this extension
* support MI protocol (just start gdb then execute `interpreter-exec mi 'show version'`)

Then this extension would work.
